contrib/bradfitz/gomemcache/memcache/memcache.go

```diff
@@ -30,27 +31,44 @@ func init() {
 	telemetry.LoadIntegration(componentName)
 }
 
+// New initializes the memcache.Client and wraps it so that all requests are traced.
+func New(servers []string, opts ...ClientOption) *Client {
+	c := memcache.New(servers...)
+	return wrapClient(c, servers, opts...)
+}
+
 // WrapClient wraps a memcache.Client so that all requests are traced using the
 // default tracer with the service name "memcached".
+// Deprecated: use New instead.
 func WrapClient(client *memcache.Client, opts ...ClientOption) *Client {
+	return wrapClient(client, nil, opts...)
+}
+
+func wrapClient(client *memcache.Client, servers []string, opts ...ClientOption) *Client {
 	cfg := new(clientConfig)
 	defaults(cfg)
 	for _, opt := range opts {
 		opt(cfg)
 	}
 	log.Debug("contrib/bradfitz/gomemcache/memcache: Wrapping Client: %#v", cfg)
-	return &Client{
+
+	wClient := &Client{
 		Client:  client,
 		cfg:     cfg,
 		context: context.Background(),
 	}
+	if len(servers) > 0 {
+		wClient.serverList = strings.Join(servers, ",")
+	}
+	return wClient
 }
 
 // A Client is used to trace requests to the memcached server.
 type Client struct {
 	*memcache.Client
-	cfg     *clientConfig
-	context context.Context
+	cfg        *clientConfig
+	context    context.Context
+	serverList string
```

Comment:
`serverList` should probably live in `cfg` with the other values that go into span tags.

Also nit:
```suggestion
	servers string
```